src/tools/rbd_mirror/image_map/RemoveRequest.h

```diff
@@ -0,0 +1,62 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
It kind of felt a bit odd to accept a map of `images -> data` to be updated and a set of image ids to remove, so I made them into separate state machines (earlier I did have them clubbed up as they were accepting a map of `image -> instance ids` with empty instance_id hinting removal).